app/server.go

```diff
@@ -236,6 +267,13 @@ func Migrate(c *Config) error {
 	if err != nil {
 		return err
 	}
+
+	db := store.DB()
+	auditRepository := audit_repos.NewPostgresRepository(db)
```

Comment:
can just simply do like this?
```suggestion
	auditRepository := audit_repos.NewPostgresRepository(store.DB())
```